externs/shaka/player.js

```diff
@@ -970,6 +971,8 @@ shaka.extern.xml.Node;
  *   but may raise issues if manifest does not have stream consistency
  *   between periods.
  *   Defaults to <code>false</code>.
+ * @property {number} updatePeriod
+ *   Override the update time for the manifest
```

Comment:
Please add a better description and put the default value.

Reply:
Description updated with proper manifest naming and default value